psql2mysql/__init__.py

```diff
@@ -339,12 +342,16 @@ def do_prechecks(config, source, target):
                 output_table.add_row([', '.join(item["primary"]),
                                       item['column']])
             print(output_table)
-            print("Error during prechecks. "
-                  "Too long text values found in the source database.")
+            print("Too long text values found in the source database.")
             prechecks_ok = False
 
     if prechecks_ok:
         print("Success. No errors found during prechecks.")
+    else:
+        raise Psql2MysqlRuntimeError(
+            "Errors were found during prechecks. Please address the reported "
+            "issues manually and try again."
+        )
```

Comment:
I still unsure if it might be better to exit upon first problem found. Instead of collection all problems of the database at once.

Reply:
May be.
On the other hand, if you have many small errors that are easy to fix, it might be better to fix of them before running the tool again.

Reply:
Yeah.  I'll just leave it as is for now.